spring-boot-docs/src/main/asciidoc/appendix-application-properties.adoc

```diff
@@ -787,6 +787,7 @@ content into your application; rather pick only the properties that you need.
 	spring.redis.cluster.max-redirects= # Maximum number of redirects to follow when executing commands across the cluster.
 	spring.redis.cluster.nodes= # Comma-separated list of "host:port" pairs to bootstrap from.
 	spring.redis.database=0 # Database index used by the connection factory.
+	spring.redis.url=redis://user:password@example.com:6379 # Connection URL, will override host, port and password (user will be ignored)
```

Comment:
Leaving the user portion blank (of userinfo) is the recommended way how to represent a Redis URI with credentials for now (`redis://:password@…`). Some clients accept `redis://password@…` but I'd rather stay on the safe side here.

See also https://www.iana.org/assignments/uri-schemes/prov/redis